src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public static String getRandomColor() {
+        int randomIndex = new Random().nextInt(Color.values().length);
+        Color color = Color.values()[randomIndex];
+        return color.toString();
     }
```

Comment:
According to the checklist, static methods are generally considered bad practice. Instead of using a static method, you should create an instance of the class and use its methods.